content/AppleAuth gem integration with Ruby on Rails - Antonieta Alvarez.md

```diff
@@ -0,0 +1,56 @@
+# AppleAuth gem, Rails Apple Sing In integration
+
+<img src="images/apple_auth.png" alt="Apple" />
+
+## What is AppleAuth?
+
+[AppleAuth](https://rubygems.org/gems/apple_auth) is a rootstrap developed Ruby gem to integrate Apple Sign In in our server side applications. 
+
+## Why implement apple sign in in our projects?
+
+This year, on the last iOS 13 release, "Sign in with Apple" login was one of the new features. At the same time the announce and update his App Store Review Guideline that if your ios app implements a third-party or social login service, like login with Facebook, will be mandatory to offer apple sign in as one option, by the end of April 2020.  Besides following the standards, implement this user authentication will let sign in using their two-factor authentication Apple ID. After the user follows Sign in with Apple to log in, your app receives tokens and user information that you can use to authenticate the user in your server.
+
+### Apple sign-in workflow:
```

Comment:
I would move this section to the top changin the title to `What it is Sign in with Apple?` to first explain it a little and show the workflow

Reply:
I would remove the colon (:) and just leave the title here